### PR TITLE
lua: adds test about ScFlowVarGet

### PR DESCRIPTION
**Pull request by catenacyber**
cf redmine 2094
https://redmine.openinfosecfoundation.org/issues/2094

cc @jufajardini

### CODE REVIEW

**Comment by catenacyber** [comment]
@jufajardini what do you think of this PR ?

File: tests/lua-scflowvarget/test.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    - HAVE_LUA
+
+args:
+ - -k none
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: alert
```
Comment (jufajardini) [inline tests/lua-scflowvarget/test.yaml:13]: I was thinking if we could add TestVar to the check here:
```suggestion
        event_type: alert
        metadata.flowvars[0].TestVar: "/zib100/zib100.json?origin=orf.at HTTP/1.1"
```

To make it more explicit/illustrative

**Review by jufajardini — COMMENTED** [review]
Sorry for not reviewing it before, didn't realize I should. :x

I like it, I think it also does a good job as an example on how to approach such cases.

Did just one suggestion, with that in mind.

File: tests/lua-scflowvarget/test.yaml (hunk 1 above)
Comment (catenacyber) [reply tests/lua-scflowvarget/test.yaml:13]: Thanks

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata-verify/pull/580

**Pull request closed without merging**